### PR TITLE
(MAIN) fill README & MAINTAINERS

### PR DESCRIPTION
**Pull request by kevpl**
[skip ci]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,7 +136,31 @@ you'd like to provide your own hosts file, set the `CONFIG` environment variable
 Acceptance tests will also fail by default, for the same reason given above as
 spec tests.
 
-## 5. Publish your library!
+## 5. Create documentation
+
+In our experience, the better the documentation, the easier to use a library
+usually is. A beginner version of documentation can simply link to the
+Rubydocs that will be generated automatically (example from 
+[baeker-pe](https://github.com/puppetlabs/beaker-pe):
```
Comment (tvpartytonight) [inline README.md]: looks like a typo.

File: README.md [hunk 2]
```diff
@@ -136,7 +136,31 @@ you'd like to provide your own hosts file, set the `CONFIG` environment variable
...
+
+```
+- [Rubydocs](http://www.rubydoc.info/github/puppetlabs/beaker-pe) contain the
```
Comment (tvpartytonight) [inline README.md:137]: I think the `` ``` `` makes this monospaced and doesn't render the `[Rubydocs](http:www...)` correctly.

**Comment by kevpl** [comment]
alright, think I got it. Thanks for the first look @tvpartytonight, and I'm ready for another!

**Comment by tvpartytonight** [comment]
LGTM!

**Pull request merged**